analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Common/Checks/ByteArrayCheck.cs

```diff
@@ -22,9 +22,9 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using SonarAnalyzer.Helpers;
-using SonarAnalyzer.SymbolicExecution.Common.Constraints;
+using SonarAnalyzer.SymbolicExecution.Constraints;
 
-namespace SonarAnalyzer.SymbolicExecution.Common.Checks
+namespace SonarAnalyzer.SymbolicExecution.Checks
```

Comment:
I am a bit confused - shouldn't this change be followed by moving these Checks to a different folder - `analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Checks` ?

Reply:
nevermind, this is done in #5087